azurerm/internal/services/netapp/resource_arm_netapp_volume.go

```diff
@@ -89,6 +89,18 @@ func resourceArmNetAppVolume() *schema.Resource {
 				ValidateFunc: azure.ValidateResourceID,
 			},
 
+			"protocol_types": {
```

Comment:
Same as above comment

Reply:
This name aligns with all other client tools, e.g. SDKs, ARM Templates, Portal, Powershell and CLI so this name will be very familiar to the users.